### ISSUE TITLE
Uppaal 5.1.0-beta2: LanguageServer closed connection upon opening Uppaal for the first time

### ISSUE THREAD

**Issue by magoorden** [UPPAALModelChecker/UPPAAL-Meta#224]
**Describe the bug**
I just downloaded the latest Uppaal beta (5.1.0-beta2) and openend it on my MacBook. I immediately got a `LangeuageServer closed connection` error. That would not be a good first impression to new users.

The full exception trace:

```
java.lang.RuntimeException: LanguageServer closed connection.
	at com.uppaal.language.LanguageServer.instance(LanguageServer.java:52)
	at as.a(Unknown Source)
	at aN.propertyChange(Unknown Source)
	at java.desktop/java.beans.PropertyChangeSupport.fire(PropertyChangeSupport.java:343)
	at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:335)
	at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:268)
	at by.put(Unknown Source)
	at as.b(Unknown Source)
	at as.g(Unknown Source)
	at com.uppaal.gui.Main.initialize(Unknown Source)
	at com.uppaal.gui.Main.lambda$main$0(Unknown Source)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: LanguageServer closed connection.
	at com.uppaal.language.Protocol.handshake(Protocol.java:48)
	at com.uppaal.language.LanguageServer.checkAvailability(LanguageServer.java:176)
	at com.uppaal.language.LanguageServer.instance(LanguageServer.java:48)
	... 23 more
```

**To Reproduce**
1. Download Uppaal 5.1.0-beta2.
2. Open Uppaal.

**Expected behavior**
No error upon opening the tool.

**Version(s) of UPPAAL tested**
Uppaal 5.1.0-beta2.

**Desktop (please complete the following information):**
 - Version: UPPAAL 5.1.0-beta2 (rev. 9376C6E11D2292F9), 2023-09-24
 - OS, Java version: macOS 14.0 (23A344)

**Comment by thorulf4** [comment]
Thanks for the bug report, the autocomplete feature is currently unstable. If this problem persists you can disable autocomplete in edit->preferences changes may require restarting uppaal to take effect

**Comment by magoorden** [comment]
> Thanks for the bug report, the autocomplete feature is currently unstable. If this problem persists you can disable autocomplete in edit->preferences changes may require restarting uppaal to take effect

I also get the error `Language server must be initialized before use` when I want to edit guards. So the unstable autocomplete makes it currently infeasable to work with a model. 

Even though I disabled the autocomplete setting in `Preferences`, and restarted the application, the `LangeuageServer closed connection` persists.

**Comment by thorulf4** [comment]
Sorry for the inconvenience I'll look into fixing this

**Comment by thorulf4** [comment]
Hey I think I've found the cause behind your issues. It turns out the language server doesn't work when Uppaal is installed on a path with a space in it. When you have time could you check if your Uppaal was installed on a path with a space, so I know if I can close this issue or not.

**Comment by magoorden** [comment]
> Hey I think I've found the cause behind your issues. It turns out the language server doesn't work when Uppaal is installed on a path with a space in it. When you have time could you check if your Uppaal was installed on a path with a space, so I know if I can close this issue or not.

This is the full path to the Uppaal app-folder on my MacBook:

`MacintoshHD/Applications/UPPAAL-5.1.0-beta2.app`

So after chaning `Macintosh HD` to `MacintoshHD` the error message is still there.

**Comment by magoorden** [comment]
The issue is solved in the latest version UPPAAL 5.1.0-beta3. No connection errors with or without the autocomplete setting enabled.
